Modules/Quotas/src/Quotas.Infrastructure.Database.Postgres/Migrations/20230601142535_TierQuotas.cs

```diff
@@ -36,21 +36,21 @@ protected override void Up(MigrationBuilder migrationBuilder)
                 columns: table => new
                 {
                     Id = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: false),
-                    _definitionId = table.Column<string>(type: "character(20)", unicode: false, fixedLength: true, maxLength: 20, nullable: true),
-                    ApplyTo = table.Column<string>(type: "character(36)", nullable: true),
+                    IdentityAddress = table.Column<string>(type: "character(36)", nullable: true),
```

Comment:
There is the IdentityAddress again :(

It's because you removed
```c#
builder.HasMany<TierQuota>(x => x.TierQuotas).WithOne().HasForeignKey(x => x.ApplyTo);
```
from IdentityEntityTypeConfiguration